tests/transform_dialect/cpu/matmul.mlir

```diff
@@ -21,7 +21,7 @@ func.func @matmul_static(
 // RUN: FileCheck %s --check-prefix=CODEGEN-CUSTOM-DISPATCH-FORMATION
 
 // CODEGEN-CUSTOM-DISPATCH-FORMATION: hal.executable private @matmul_static_dispatch_0 {
-// CODEGEN-CUSTOM-DISPATCH-FORMATION:   hal.executable.variant public @embedded_elf_x86_64, target = #executable_target_embedded_elf_x86_64_ {
+// CODEGEN-CUSTOM-DISPATCH-FORMATION:   hal.executable.variant public @embedded_elf_{{[[:alnum:]_]+}}, target = #executable_target_embedded_elf_{{[[:alnum:]_]+}} {
```

Comment:
this test seems to be a change detector (https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html)

since this test is not setting the target at you can remove everything after the hal.variant (otherwise if the default is not embedded_elfs it'll fail too)
```suggestion
// CODEGEN-CUSTOM-DISPATCH-FORMATION:   hal.executable.variant
```

Reply:
yes, +1, thanks!

Just drop all spurious attributes from the test